Content.Server/Ame/Components/AmeControllerComponent.cs

```diff
@@ -72,10 +72,23 @@ public sealed partial class AmeControllerComponent : SharedAmeControllerComponen
     [DataField("nextUpdate")]
     public TimeSpan NextUpdate = default!;
 
+    /// <summary>
+    /// The next time this will try to update the controller UI.
+    /// </summary>
+    [DataField("nextUIUpdate")]
```

Comment:
Data field tags don't need to be specified anymore. They are derived from the field or property that they are on, with the first letter automatically made lowercase.

```suggestion
    [DataField]
```

Reply:
since this does not need to be referenced anywhere and instead threw an error during integration testing, I just removed the DataField part